### PR TITLE
Docs for october 4960

### PR DESCRIPTION
**Pull request by GinoPane**
Docs update for https://github.com/octobercms/october/pull/4960

### CODE REVIEW

**Comment by LukeTowers** [comment]
@GinoPane this could probably just be a section under the "themes/development" page

**Comment by GinoPane** [comment]
@LukeTowers ok, I moved it

File: themes-development.md [hunk 1]
```diff
@@ -136,3 +137,40 @@ A theme can depend on plugins by defining a **require** option in the [Theme inf
...
+            lang.php      <=== Localization file
+          fr/
+            lang.php
+
+
+The **lang.php** file should define and return an array of any depth, for example:
+
+    <?php
+
+    return [
```
Comment (LukeTowers) [inline themes-development.md:169]:
```suggestion
    <?php return [
```

File: themes-development.md [hunk 2]
```diff
@@ -136,3 +137,38 @@ A theme can depend on plugins by defining a **require** option in the [Theme inf
...
 
 When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.
+
+<a name="localization"></a>
+## Localization
+
+Themes can have localization files in the **lang** subdirectory of the theme directory. Theme's localization files
+ are registered automatically when the theme is being edited in the backend. The localization strings are supported
+  automatically in the back-end customization form labels - if you provide the localization key instead of a real
+   string, the system will try to load it from the localization file.
```
Comment (LukeTowers) [inline themes-development.md:147]:
```suggestion
Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
```

File: themes-development.md [hunk 3]
```diff
@@ -136,3 +137,35 @@ A theme can depend on plugins by defining a **require** option in the [Theme inf
         - Acme.Blog
 
 When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.
+
+<a name="localization"></a>
+## Localization
+
+Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
+   
+> **Note**: For translating front-end content, [there are plugins that can be used](http://octobercms.com/plugin/rainlab-translate) for this purpose.
```
Comment (LukeTowers) [inline themes-development.md:146]:
```suggestion
> **Note**: Translating frontend content should be handled with the [RainLab.Translate](https://octobercms.com/plugin/rainlab-translate) plugin.
```

File: themes-development.md [hunk 4]
```diff
@@ -136,3 +137,35 @@ A theme can depend on plugins by defining a **require** option in the [Theme inf
...
+The **lang.php** file should define and return an array of any depth, for example:
+
+    <?php return [
+        'options' => [
+            'website_name' => 'OctoberCMS'
+        ]
+    ];
+
+> **Note**: the full localization key format is the following: theme.theme-code::lang.key. So for example above it
+> would be `theme.theme-code::lang.options.website_name`
```
Comment (LukeTowers) [inline themes-development.md:171]:
```suggestion
You are then able to reference the keys using `themes.theme-code::lang.key`. In the above example, the full language key you would use to reference the "website_name" localization key would be `themes.acme::lang.options.website_name`
```

File: themes-development.md [hunk 5]
```diff
@@ -136,3 +137,34 @@ A theme can depend on plugins by defining a **require** option in the [Theme inf
...
 When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.
+
+<a name="localization"></a>
+## Localization
+
+Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
+   
+> **Note**: Translating frontend content should be handled with the [RainLab.Translate](https://octobercms.com/plugin/rainlab-translate) plugin.
+
+<a name="file-structure"></a>
```
Comment (LukeTowers) [inline themes-development.md:148]:
```suggestion
<a name="localization-file-structure"></a>
```

**Pull request merged**